cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -2128,4 +2153,24 @@ mod tests {
         .unwrap_err();
         assert!(matches!(result, CoreEnvironmentError::PackageNotFound(_)));
     }
+
+    #[test]
+    fn test_get_install_ids_to_uninstall_with_version() {
+        let mut manifest_mock = generate_mock_manifest(vec![("testInstallID", "dotted.package")]);
+
+        if let TypedManifest::Catalog(ref mut catalog) = manifest_mock {
+            if let ManifestPackageDescriptor::Catalog(descriptor) =
+                catalog.install.get_mut("testInstallID").unwrap()
+            {
+                descriptor.version = Some("1.0".to_string());
+            };
+        }
+
+        let result = CoreEnvironment::get_install_ids_to_uninstall(&manifest_mock, vec![
+            "dotted.package@1.0".to_string(),
+        ])
+        .unwrap();
+
+        assert_eq!(result.len(), 1);
```

Comment:
nit: assert install_id as well?

Reply:
Do you mean `iid@version`?

At least the ticket didnt ask for uninstalling by iid + version.
My understanding was that we support this more of a convenience where you still have the `pkgpath@version` but in a clipboard or shell history.

Reply:
Oh I just meant this nit
```suggestion
        assert_eq!(result.len(), 1);
        assert_eq!(result[0], "testInstallID");
```